app/javascript/components/search/StudySearchProvider.js

```diff
@@ -19,17 +30,36 @@ export const StudySearchContext = React.createContext(emptySearch)
 export default class StudySearchProvider extends React.Component {
   constructor(props) {
     super(props)
+    const queryParams = queryString.parse(window.location.search);
     let initialState = _cloneDeep(emptySearch)
```

Comment:
What is this cloneDeep?

Reply:
It's a lodash method to create a deep copy of an object.  Makes sure that we get a full copy of the object and not sharing any references